software/horilla.yml

```
@@ -0,0 +1,15 @@
+name: "Horilla"
+website_url: "https://www.horilla.com"
+source_code_url: "https://github.com/horilla-opensource/horilla"
+description: "Horilla is a Free and Open Source HRMS (Human Resource Management System) Software designed to streamline HR processes and enhance organizational efficiency."
+licenses:
+  - LGPL-2.1 license
+platforms:
+  - Python
+  - Docker
```

> https://www.horilla.com/docs

I cant seem to find Docker
```suggestion
platforms:
  - Python
```

---

We are awaiting approval for reclaiming the username Horilla in github due to some privacy reasons, but the images of Horilla are available by other users in Docker hub currently.

---

Are there currently working installation instructions for Docker? If not then this should be removed.

---

@nikhilhorilla pinging

---

Hi @Ki-er ,
Th docker image is now up in our docker hub account. 
https://hub.docker.com/r/horilla/horilla

---

There is still no working installation instruction for Docker though over at https://www.horilla.com/docs/installation. Due to this, Docker should be removed.